Dockerfile

```diff
@@ -10,16 +10,26 @@ RUN apk --update add ca-certificates
 FROM alpine:3.16.0 as directories
 RUN mkdir /etc/otel/
 
+FROM debian:11.3 as systemd
+RUN apt update && apt install -y systemd
+# prepare package with journald and it's dependencies
+# h stands for dereference of symbolic links
+RUN tar czhf journalctl.tar.gz /bin/journalctl $(ldd /bin/journalctl | grep -oP "\/.*? ")
+# extract package to /output so it can be take as base for scratch image
+RUN mkdir /output && tar xf /journalctl.tar.gz --directory /output
```

Comment:
Why are we compressing these files, if we immediately decompress them in the same build stage?

Reply:
this is copying only required files while preserving paths and meantime skipping symlinks. pretty easy to explain :joy:

Reply:
Maybe at least leave a comment so it's clear why this exists, if not how it works exactly?